test/test_UKF/main.cpp

```diff
@@ -95,7 +95,7 @@ int main()
 
     /* Step 2.2 - Define the prediction step. */
 
-    /* Initialize the particle filter prediction step and pass the ownership of the state model. */    
+    /* Initialize the particle filter prediction step and pass the ownership of the state model. */
```

Comment:
I think this is my fault, I wrote this test  and this comment is wrong since the `UKF` has nothing to do with particles. :laughing: 

Can we change this to `Initialize the unscented Kalman filter prediction step and pass the ownership of the state model`. Probably it is better to do this in another commit since unrelated to the description of this one. What do you think?

Reply:
Done!

Reply:
Thanks :+1: